soccer/src/soccer/soccer_mom.cpp

```diff
@@ -0,0 +1,20 @@
+#include <rclcpp/rclcpp.hpp>
+#include "soccer_mom.hpp"
+
+
```

Comment:
Same comment as above. It is extremely important on the C++ aspect of the code base as we don't have too much documentation on it the first place.